ultradns/common_test.go

```diff
@@ -1,13 +1,157 @@
 package ultradns
 
 import (
+	"encoding/json"
 	"fmt"
+	"testing"
 
-	"github.com/terra-farm/udnssdk"
+	udnssdk "github.com/aliasgharmhowwala/ultradns-sdk-go"
```

Comment:
No personal for reference please.

Reply:
yup will do it once we merge the sdk plugin to master in ultradns project